Documentation/gettingstarted/encryption.rst

```diff
@@ -200,6 +201,58 @@ used by Cilium. The key variable is a uint8 with value between 0-16 and should
 be monotonically increasing every re-key with a rollover from 16 to 0. The
 Cilium agent will default to KEYID of zero if its not specified in the secret.
 
+.. _encryption_ipam_eni:
+
+Encryption with IPAM ENI mode
+==============================
+
+In order to enable encryption on EKS with the IPAM ENI mode, deploy cilium via
+Helm with the following options:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set ipam.mode=eni \\
+      --set tunnel=disabled \\
+      --set encryption.enabled=true \\
+      --set encryption.nodeEncryption=false
```

Comment:
I'll defer to @jrfastab here but from what I understand, the user must also specify `--ipv4-pod-subnets` to cover all subnets that AWS may allocate IPs from within the cluster. Note this is not currently exposed in the helm charts, so users would need to either use `extraArgs` or we'd need to introduce dedicated helm options for this setting.